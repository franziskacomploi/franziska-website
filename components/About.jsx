import React from 'react'

const About = () => {
  return (
    <div className="bg-beige pt-28 pb-44">
      <div className="mx-auto w-1/2">
        <div className="headline-xl text-center md:text-left my-4">
          About Me
        </div>
        <div className="content-big text-center md:text-left font-light mb-4">
          {`Hi, I'm Fran!
            The year 2020 probably held a huge transition for a lot of us. Same for me. I used to work as a Content Manager and now turned into a Software Developer with a passion for building amazing products.
            My motivation comes from tasks fueled by passion and big dreams! When I set myself the goal to learn something new, I got myself a job in only 3 months. And I'm still learning new things everyday.
          `}
        </div>
      </div>
    </div>
  )
}

export default About
