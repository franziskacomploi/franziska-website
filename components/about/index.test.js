import { mount } from '@vue/test-utils';
import { it, expect, describe } from 'vitest';
import About from './index.vue';

describe('About', () => {
  it('should mount', () => {
    const wrapper = mount(About, { shallow: true });
    expect(wrapper.exists()).toBeTruthy();
  });

  it('should render correctly', () => {
    const wrapper = mount(About, { shallow: true });
    expect(wrapper.element).toMatchSnapshot();
  });
});
