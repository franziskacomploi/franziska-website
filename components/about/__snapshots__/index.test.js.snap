// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`About > should render correctly 1`] = `
<div
  class="bg-cultured pt-16 md:pt-28 pb-44"
>
  <div
    class="mx-auto px-8 md:w-1/2"
  >
    <div
      class="headline-xl text-center md:text-left my-4"
    >
      About me
    </div>
    <div
      class="content-big text-center md:text-left font-light mb-4"
    >
      Hi, I'm Fran! The year 2020 probably held a huge transition for a lot of us and it was no different for me. After I finished my universtity in 2018, I moved to Berlin and worked in Marketing at different Start-ups. A few years later I was rethinking my career and dedicated myself to learn coding. After only 3 months, I got myslef a new job. Now I'm focusing on cloud, data & engineering, as well as sustainabable approaches.
    </div>
  </div>
</div>
`;
